src/elements/Modal/Modal.styles.ts

```diff
@@ -0,0 +1,60 @@
+import { StyleSheet } from 'react-native';
+import * as colors from '@util/colors';
+
+export default StyleSheet.create({
+	wrapper: {
+		width: '100%',
+		height: '100%',
+		alignItems: 'center',
+		justifyContent: 'center',
+	},
+	underlay: {
+		width: '100%',
+		height: '100%',
+		position: 'absolute',
+		backgroundColor: colors.WHITE_TRANSPARENT,
+	},
+	container: {
+		width: 'calc(100% - 15px)',
+		height: 'fit-content',
+		position: 'absolute',
+		alignItems: 'center',
+		overflow: 'hidden',
+		borderRadius: 10,
+		shadowColor: 'rgba(0, 0, 0, 0.25)',
+		shadowOffset: {
+			width: 0,
+			height: 4,
+		},
+		shadowRadius: 4,
+	},
+	header: {
+		width: '100%',
+		height: 80,
+		alignItems: 'center',
+		justifyContent: 'center',
+		backgroundColor: colors.BANANA_YELLOW,
```

Comment:
This color should be variable. This would allow parent components to set the value. For example, an alert modal could use this modal but set the BG to red.

Should this component accept a header color prop? Or a modal 'type' prop that is a limited number of options that map to different colors (primary, accent, default, disabled, etc.).

Reply:
I like both ideas! But I am leaning more towards passing the type instead because, the way I think about it, the parent should not have to worry about what the color of the modal should have, but rather the type of modal it wants to render. Though, I do not see any crippling disadvantages on doing it the other way.

Either way, we can use the values of either props for conditional rendering and display the appropriate data!